.nvmrc

```
@@ -0,0 +1 @@
+14.8.0
```

```suggestion
14
```
that way it’ll use the latest version, 14.10.1 atm